src/components/dashboard/dashboard.component.tsx

```diff
@@ -314,7 +316,9 @@ function FormsList({ forms, isValidating, mutate, t }: FormsListProps) {
     name: (
       <ConfigurableLink
         className={styles.link}
-        to={`${window.getOpenmrsSpaBase() + `form-builder/edit/` + form?.uuid}`}
+        to={editSchemaUrl}
+        templateParams={{ formUuid: form?.uuid }}
+        onMouseEnter={() => void preload(`/ws/rest/v1/form/${form?.uuid}?v=full`, openmrsFetch)}
```

Comment:
New syntax to me... what does the `void` do here? Also probably should be something like:

```suggestion
        onMouseEnter={() => form?.uuid && void preload(`/ws/rest/v1/form/${form.uuid}?v=full`, openmrsFetch)}
```

Although... can form be undefined here?

Reply:
The `preload` call was getting flagged by the [@typescript-eslint/no-misused-promises](https://typescript-eslint.io/rules/no-misused-promises/)  ESLint rule which is intended to catch unhandled rejections and missing awaits. The [void](https://typescript.tv/new-features/the-void-operator-in-typescript-and-javascript/) operator is useful when you want to suppress the return value of a function or initiate a Promise without handling its resolution. Alternatively, I could tack on a `catch` block to handle rejections.

The `form` object should always have a UUID, I think. As long as it's saved in the database. What might be missing is the associated clobdata.